Rakefile

```diff
@@ -1,7 +1,14 @@
+# coding: utf-8
 require 'html-proofer'
+require 'jekyll'
 
-task :test do
-  sh "bundle exec jekyll build"
+task :build do
+  puts 'Building site...'.bold
+  Jekyll::Commands::Build.process(profile: true)
+end
+
+task :test => [:clean, :build] do
```

Comment:
Do you think we should depend test task on other tasks?

Reply:
Yes, because in order to test you need to build first.